prow/cmd/invitations-accepter/BUILD.bazel

```diff
@@ -0,0 +1,58 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
+load("//prow:def.bzl", "prow_image")
+
+NAME = "invitations-accepter"
+
+prow_image(
+    name = "image",
+    base = "@git-base//image",
```

Comment:
I don't think we need git, the other components that don't need it use `@alpine-base//image`